api/server/volume.go

```diff
@@ -63,7 +66,10 @@ func (vd *volAPI) getConn() (*grpc.ClientConn, error) {
 		var err error
 		vd.conn, err = grpcserver.Connect(
 			vd.sdkUds,
-			[]grpc.DialOption{grpc.WithInsecure()})
+			[]grpc.DialOption{
+				grpc.WithInsecure(),
+				grpc.WithDefaultCallOptions(grpc.MaxCallRecvMsgSize(maxMsgSize)),
```

Comment:
Should we also set the `MaxCallSendMsgSize`?

K8's fix for this also includes Send:
https://github.com/kubernetes/kubernetes/pull/63894/files

Reply:
I'm not sure if we ever send that much data into the server. Let me know if you know

Reply:
Also, that is the server, we are affecting the client. I'd like not to affect the server unless needed.

Reply:
I interpreted it as the grpc server sending the results of `pxctl v l` over the grpc steam back to the client. But I just looked at the `MaxCallRecvMsgSize` documentation. Looks good.